docs/source/concepts/MatrixWorkspace.rst

```diff
@@ -8,57 +8,53 @@ Matrix Workspace
 
--  :ref:`Workspace2D <Workspace2D>` - A workspace for holding two       dimensional data in memory, this is the most commonly used workspace.
--  :ref:`EventWorkspace <EventWorkspace>` - A workspace that retains the       individual neutron event data.            
+-  :ref:`Workspace2D <Workspace2D>` - A workspace for holding 2D, acuumulated data in memory, this is the most commonly used to store histograms.
+-  :ref:`EventWorkspace <EventWorkspace>` - A workspace that retains the individual neutron event data.            
 
-What information is in a MatrixWorkspace
-----------------------------------------
-
-All Matrix Workspaces contain:
+What information is in a Matrix Workspace
+------------------------------------------
 
--  Measured or derived data with associated errors, this is referenced as a 2D array of counts and error data.  The axes are commonly "SpectraNumber" and another unit of measure, but are very flexible.
+All Matrix Workspaces contain one or more rows of data. A single row is formed of a set of arrays (Y, E, X). For example, the output of a single detector in a white-beam measuring wavelength would form a single row. In this scenario The x-axis (or "horizontal axis") has units of length for wavelength, and Y would be the counts at each wavelength. The rows may have no correlation to each other. However, they are usually marked and sorted by some other attribute, such as detector number, or scattering angle. We call this row axis the "vertical axis". Axis have :ref:`Units <Unit Factory>`.  
 
 Also they may contain:
 
--  Axes with
-   :ref:`Units <Unit Factory>`
 -  :ref:`Sample` and sample environment data
 -  :ref:`Run` logs
--  A full :ref:`instrument <instrument>` geometric definition, along with
-   an instrument parameter map
--  A spectra - detector mapping
--  A distribution flag
+-  Usable geometry may be present, for example to determine the position of each detector, see :ref:`here <InstrumentAccessLayers>` for more information
+-  Spectra - detector mapping, and other spectra level information see :ref:`here <InstrumentAccessLayers>`.
+-  :ref:`Histogram <HistogramData>` information
 -  A list of 'masked' bins
 
 Working with Matrix Workspaces in Python
 ----------------------------------------
 
-MatrixWorkspace is an abstract description of an specific workspace implementation. It provides access to a common way of accessing the data for a 2D workspace without needing to know the details of how that data is actually stored.
+As alluded to above, Matrix Workspace is an abstraction. Matrix Workspace provides access to the majority of the common information for all 2D workspaces in its family, without needing to know all the details of how that data is actually stored.
```

Comment:
Should be "MatrixWorkspace" rather than "Matrix Workspace".

Reply:
@peterfpeterson I deliberately changed from`MatrixWorkspace`, to `Matrix Workspace` but I think we should agree a common way that we present "concepts" in plain english to users who are not necessarily going to be interested in our `c++` or `python` conventions. `MatrixWorkspace` feels less natural

Reply:
I am with Pete here -- `Matrix Workspace` does not make any sense in English (or at least no one will see the connection to what our `MatrixWorkspace` is). If we add spaces to many of our class names it will be a big mess, no one will know what to search for, and it is not clear to anyone which concept is a class and which is not.